src/components/generic-elements/download-button/download-button.js

```diff
@@ -1,5 +1,31 @@
 import './download-button.scss'
+import { createRef, useState } from 'react'
```

Comment:
We tried and tried to come up with a good way to test this component, but couldn't find a good way.
Things we'd like to test (after someone clicks the button):
- On Click was fired
- New href is generated with key and expires
- download file name is correct

Because it contains an async call and uses a ref to click itself, we couldn't find a good way to obtain and check the final state. Thoughts welcome.